homeassistant/components/airzone/__init__.py

```diff
@@ -26,17 +26,20 @@ def __init__(
         coordinator: AirzoneUpdateCoordinator,
         entry: ConfigEntry,
         system_zone_id: str,
-        zone_name: str,
+        zone_data: dict,
```

Comment:
Please make the dictionary typing more specific

Reply:
Fixed in 66bd22912bbcf6577569050a66093a5dd4394fab